packages/git/src/node/dugite-git.ts

```diff
@@ -548,7 +548,7 @@ export class DugiteGit implements Git {
                 [CommitPlaceholders.SHORT_HASH, ...CommitDetailsParser.DEFAULT_PLACEHOLDERS.slice(1)] : CommitDetailsParser.DEFAULT_PLACEHOLDERS;
         args.push(...['--name-status', '--date=unix', `--format=${this.commitDetailsParser.getFormat(...placeholders)}`, '-z', '--']);
         if (options && options.uri) {
-            const file = Path.relative(this.getFsPath(repository), this.getFsPath(options.uri));
+            const file = Path.relative(this.getFsPath(repository), this.getFsPath(options.uri)) || ".";
```

Comment:
This should be `|| '.'` instead of `|| "."`.
From CI:
```
@theia/task: [compile] theiaext compile exited with code 0
217{ Error: Command failed: yarn run prepare
218error Command failed with exit code 1.
219error Command failed with exit code 1.
220
221$ yarn run clean && yarn run build
222$ theiaext clean
223$ theiaext build
224[lint] 
225[lint] ERROR: C:/projects/theia/packages/git/src/node/dugite-git.ts[553, 100]: " should be '
226[lint] 
227[lint] theiaext lint exited with code 2
228[compile] theiaext compile exited with code 0
229info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
230info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
231
232    at Promise.all.then.arr (C:\projects\theia\node_modules\lerna\node_modules\execa\index.js:236:11)
233    at <anonymous>
234    at process._tickCallback (internal/process/next_tick.js:188:7)
235  code: 1,
236  killed: false,
237  stdout: '$ yarn run clean && yarn run build\n$ theiaext clean\n$ theiaext build\n[lint] \n[lint] ERROR: C:/projects/theia/packages/git/src/node/dugite-git.ts[553, 100]: " should be \'\n[lint] \n[lint] theiaext lint exited with code 2\n[compile] theiaext compile exited with code 0\ninfo Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.\ninfo Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.\n',
238  stderr: 'error Command failed with exit code 1.\nerror Command failed with exit code 1.\n',
239  failed: true,
240  signal: null,
241  cmd: 'yarn run prepare',
242  timedOut: false,
243  exitCode: 1 }
244error Command failed with exit code 1.
245
```